src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.jsx

```diff
@@ -54,11 +62,6 @@ class SentryApplicationDetails extends AsyncView {
     return events.map(e => e.split('.').shift());
   }
 
-  handleScopeChange = (onChange, onBlur, scope, scopes, e) => {
-    onChange(scopes, e);
-    onBlur(scopes, e);
-  };
-
   onSubmitSuccess = data => {
     const {orgId} = this.props.params;
     browserHistory.push(`/settings/${orgId}/developer-settings/`);
```

Comment:
I realize this hasn't change but maybe we could add a success indicator? otherwise its a little confusing whether updating actually worked